cumulus-client/src/test/java/mil/army/usace/hec/cumulus/client/controllers/TestCumulusDssData.java

```diff
@@ -0,0 +1,40 @@
+package mil.army.usace.hec.cumulus.client.controllers;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import mil.army.usace.hec.cumulus.client.model.CumulusObjectMapper;
+import mil.army.usace.hec.cumulus.client.model.Product;
+import mil.army.usace.hec.cumulus.client.model.Watershed;
+import org.junit.jupiter.api.Test;
+
+final class TestCumulusDssData {
+
+    @Test
+    void testSerialization() throws IOException, URISyntaxException {
+        String watershedJson = readResourceFile("cumulus/json/watershed.json");
+        String productJson = readResourceFile("cumulus/json/product.json");
+        Watershed watershed = CumulusObjectMapper.mapJsonToObject(watershedJson, Watershed.class);
+        Product product = CumulusObjectMapper.mapJsonToObject(productJson, Product.class);
+        String dssPath = CumulusDssDataUtil.buildDssPath(watershed, product);
+        assertEquals("SHG/Cumberland Basin River/PRECIP/23Aug2021:0601/24Apr2022:1201/MBRFC-FORECAST", dssPath);
```

Comment:
I think those times are illegal and they dont look like they make sense. Probably better to leave them blank. Also, missing preceding and trailing "/"

Reply:
removed times and added preceding and trailing "/"